types/chartist/index.d.ts

```diff
@@ -1,7 +1,12 @@
-// Type definitions for Chartist v0.9.81
+// Type definitions for Chartist v0.11.0
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable `dt-header` in the `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `0.11`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`). It would also be great to enable as many disabled rules as you can, but that may be a lot of work.